File: App/app/src/main/java/com/example/octo_umbrella/models/Note.java
package com.example.octo_umbrella.models;

import com.google.gson.annotations.SerializedName;

public class Note {
    @SerializedName("id")
    public String Id;
    @SerializedName("userEmail")
    public String UserEmail;
    @SerializedName("title")
    public String Title;
    @SerializedName("content")
    public String Content;
    @SerializedName("createdDate")
    public String CreatedDate ;
    @SerializedName("lastModifiedDate")
    public String LastModifiedDate  ;
}

File: API/Octo-Umbrella-API/Models/Note.cs
using MongoDB.Bson.Serialization.Attributes;
using System.Text.Json.Serialization;

namespace Octo_Umbrella_API.Models
{
    public class Note
    {
        [BsonId]

        [JsonPropertyName("_id")]
        public string Id { get; set; }
        public string UserEmail { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime LastModifiedDate { get; set; }

        public Note(string userEmail, string title, string content)
        {
            Id = Guid.NewGuid().ToString();
            UserEmail = userEmail;
            Title = title;
            Content = content;
            CreatedDate = DateTime.Now;
            LastModifiedDate = CreatedDate;
        }
    }
}
